### PR TITLE
Types added

### PR DESCRIPTION
**Pull request by ziviledauderiene**

### CODE REVIEW

File: src/helpers/api.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
-import axios from 'axios';
+import axios, { AxiosResponse } from 'axios';
+import { AxiosOptions, Url } from './helpersInterfaces';
 
-export const URL = {
+export const URL: Url = {
```
Comment (JuliaShavyrova) [inline src/helpers/api.ts]: `Url` interface (and variable) name is a bit confusing, please rename it to define that it's a config for api routes

File: src/helpers/helpersInterfaces.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  observations: string;
+  current: string;
+  nowCast: string;
+  hourly: string;
+  threeHourly: string;
+  daily: string;
+  airQuality: string;
+}
+
+export interface AxiosOptions {
```
Comment (JuliaShavyrova) [inline src/helpers/helpersInterfaces.ts]: you can use AxiosRequestConfig built-in interface to define required options for your requests

Comment (ziviledauderiene) [reply src/helpers/helpersInterfaces.ts]: ok, changed. thank you

File: src/helpers/api.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
-import axios from 'axios';
+import axios, { AxiosRequestConfig, AxiosResponse } from 'axios';
+import { UrlInterface } from './helpersInterfaces';
 
-export const URL = {
+export const URL: UrlInterface = {
```
Comment (JuliaShavyrova) [inline src/helpers/api.ts]: 1. your const URL is definitely not a url, it's a config for api routes you going to use in app, so please rename, the same for interface - it's not describing a url model, it describes very different stuff
2. no need to use word `Interface/Type` when you're creating an interface/type. One of the reasons: if you'll need to transform it from interface to type in future, you'll need to rename and replace it everywhere. You can follow [this](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines) coding guideline as example

**Review by JuliaShavyrova — APPROVED** [review]

**Pull request merged**